File: object_oriented_programming/uber_practice/python/account.py
#!/usr/bin/python3
""" Practice of declaring classes and objects in Python.
    This module is for superclass Account.
    """


class Account:
    id = int
    name = str
    document = str
    email = str
    password = str

    # __init__ is the magic method to declare constructors
    def __init__(self, name, document):
        self.name = name
        self.document = document

File: object_oriented_programming/uber_practice/javascript/account.js
#!/usr/bin/node

export default class Account {
  constructor (name, document) {
    this.id;
    this.name = name;
    this.document = document;
    this.email;
    this.password;
  }

  print () {
    console.log(`Account name: ${this.name} | email: ${this.email}`);
  }
}
